app/lib/constants/settings/general.rb

```diff
@@ -99,10 +99,6 @@ def self.details
             description: I18n.t("lib.constants.settings.general.sidebar.description"),
             placeholder: I18n.t("lib.constants.settings.general.sidebar.placeholder")
           },
-          sponsor_headline: {
```

Comment:
This was being used on the admin in `app/views/admin/settings/forms/_sponsors.html.erb` which I've deleted.